File: 2132-convert-1d-array-into-2d-array/convert-1d-array-into-2d-array.cpp
class Solution {
public:
    vector<vector<int>> construct2DArray(vector<int>& ori, int m, int n) {
        vector<vector<int>> ans;
        int l=ori.size();
        if(l==n*m){
            int k=0;
            for(int i=0;i<m;i++){
                vector <int> v;
                for(int j=0;j<n;j++){
                    v.push_back(ori[k]);
                    k++;
                }
                ans.push_back(v);
            }
        }
        return ans;
    }
};

File: 2132-convert-1d-array-into-2d-array/convert-1d-array-into-2d-array.py
class Solution(object):
    def construct2DArray(self, ori, m, n):
        """
        :type original: List[int]
        :type m: int
        :type n: int
        :rtype: List[List[int]]
        """
        l=len(ori)
        ans=[]
        if l==m*n:
            k=0
            for i in range(m):
                row=[]
                for j in range(n):
                    row.append(ori[k])
                    k+=1
                ans.append(row)
            
        return ans